src/AWS/Orleans.Reminders.DynamoDB/SiloHostBuilderReminderExtensions.cs

```diff
@@ -63,7 +63,7 @@ public static IServiceCollection UseDynamoDBReminderService(this IServiceCollect
         {
             services.AddSingleton<IReminderTable, DynamoDBReminderTable>();
             services.Configure(configure);
-            services.TryConfigureFormatter<DynamoDBReminderTableOptions, DynamoDBReminderTableOptionsFormatter>();
+            services.TryConfigureFormatter<DynamoDBReminderTableOptions>();
```

Comment:
TryConfigureFormatter<TOption> won't work and should be removed.
This call should always call ConfigureFormatter.

Reply:
Same applies to all users of TryConfigureFormatter< TOptions >